packages/amplication-data-service-generator/src/server/static/src/auth/gqlDefaultAuth.guard.ts

```diff
@@ -1,3 +1,3 @@
-import { GqlBasicAuthGuard } from "./gqlBasicAuth.guard";
+import { GqlBasicAuthGuard } from "./basic/gqlBasicAuth.guard";
 
 export class GqlDefaultAuthGuard extends GqlBasicAuthGuard {}
```

Comment:
This should be dynamic

Reply:
what is your proposal? GqlDefaultAuthGuard made dynamically or extends the defaultAuthGuard?

Reply:
Like we discussed yesterday, it is best to use a single guard for GQL that extended defaultGuard that adds the "getRequest" function.
This way we only need to add a single guard file per provider, and all guards will be extended for GraphQL via the default guard.